lib/openstudio-standards/btap/bridging.rb

```diff
@@ -1502,7 +1502,7 @@ def initialize(model = nil, argh = {})
           next unless construction[:stypes  ] == stypes
           next     if construction[:surfaces].empty?
 
-          construction[:surfaces].values.each { |surface| surface.setConstruction(construction[:uo]) }
+          # construction[:surfaces].values.each { |surface| surface.setConstruction(construction[:uo]) }
```

Comment:
Not sure what this call is attempting, but commenting it out allows BTAP thermal bridging tests to pass. A revised skylight/well distribution would affect (linear) thermal bridging calculations.